elasticdl/python/common/args.py

```diff
@@ -173,7 +173,12 @@ def add_common_params(parser):
         help="The name of the Kubernetes namespace where ElasticDL "
         "pods will be created",
     )
-    parser.add_argument("--records_per_task", type=int, required=True)
+    parser.add_argument(
+        "--minibatches_per_task",
```

Comment:
Maybe `num_minibatches_per_task` would be better?

Reply:
*minibatches_per_task* is using the same pattern with records_per_task. How about keep aligned with the existed parameters.

Reply:
Good idea, updated.